src/javascripts/main.js

```diff
@@ -2,14 +2,41 @@ import firebase from 'firebase/app';
 import apiKeys from './helpers/apiKeys.json';
 
 import auth from './components/auth/auth';
-import myNavbar from './components/myNavbar/myNavbar';
 import authData from './helpers/data/authData';
+import mushroomList from './components/mushroomList/mushroomList';
+import mycologistList from './components/mycologistList/mycologistList';
+import myNavbar from './components/myNavbar/myNavbar';
 
 import '../styles/main.scss';
 
+const authDiv = $('#auth');
+const forestDiv = $('#forest');
+const hutDiv = $('#hut');
+const logoutButton = $('#navbar-logout-button');
+const singleMycoDiv = $('#single-myco');
+
+const showLoggedInView = () => {
+  authDiv.addClass('hide');
+  forestDiv.removeClass('hide');
+  hutDiv.removeClass('hide');
+  singleMycoDiv.removeClass('hide');
+  logoutButton.removeClass('hide');
+
+  mushroomList.buildForest();
+  mycologistList.buildHut();
+};
+
+const showLoggedOutView = () => {
+  authDiv.removeClass('hide');
+  forestDiv.addClass('hide');
+  hutDiv.addClass('hide');
+  singleMycoDiv.addClass('hide');
+  logoutButton.addClass('hide');
+};
+
 const init = () => {
   firebase.initializeApp(apiKeys.firebaseConfig);
-  authData.checkLoginStatus();
+  authData.checkLoginStatus(showLoggedInView, showLoggedOutView);
```

Comment:
is this another way of exporting these?

Reply:
what do you mean?

Reply:
are you essentially exporting the showLoggedInView and ShowLoggedOutView functions by passing them as arguments into authData.checkLoginStatus?

Reply:
so I'm not exporting/importing them, but I am passing them as args to the `authData.checkLoginStatus`. It's *kind of* like exporting/importing...but on a smaller, function-scale instead of a larger module-scale (does that make any sense?)

Reply:
Yeah that makes sense! Just hadn’t seen that before (I don’t think) so I was curious